LineUp v4 compile errors in OverviewColumn

* Release number or git hash: 
* Web browser version and OS: 
* Environment (local or deployed): 

### Steps to reproduce

1. See [lineupjs4 CircleCI build](https://circleci.com/gh/datavisyn/tdp_core/1569)

### Observed behavior

```
src/lineup/internal/OverviewColumn.ts:21:28 - error TS2339: Property 'setDefaultRenderer' does not exist on type 'OverviewColumn'.

21     (<OverviewColumn>this).setDefaultRenderer('boolean');
                              ~~~~~~~~~~~~~~~~~~

src/lineup/internal/OverviewColumn.ts:22:28 - error TS2339: Property 'setDefaultGroupRenderer' does not exist on type 'OverviewColumn'.

22     (<OverviewColumn>this).setDefaultGroupRenderer('boolean');
                              ~~~~~~~~~~~~~~~~~~~~~~~

src/lineup/internal/OverviewColumn.ts:23:28 - error TS2339: Property 'setDefaultSummaryRenderer' does not exist on type 'OverviewColumn'.

23     (<OverviewColumn>this).setDefaultSummaryRenderer('categorical');
```

Missing `setDefaultRenderer`, `setSummaryRenderer`, and `setDefaultSummaryRenderer`


### Expected behavior

No build errors

--- a/src/lineup/internal/OverviewColumn.ts
+++ b/src/lineup/internal/OverviewColumn.ts
@@ -16,12 +16,12 @@ export default class OverviewColumn extends BooleanColumn {
 
   constructor(id: string, desc: IBooleanColumnDesc) {
     super(id, Object.assign(desc, {
-      label: i18n.t('tdp:core.lineup.OverviewColumn.overviewSelection')
+      label: i18n.t('tdp:core.lineup.OverviewColumn.overviewSelection'),
+      renderer: 'boolean',
+      groupRenderer: 'boolean',
+      summaryRenderer: 'categorical'
     }));
-    (<OverviewColumn>this).setDefaultRenderer('boolean');
-    (<OverviewColumn>this).setDefaultGroupRenderer('boolean');
-    (<OverviewColumn>this).setDefaultSummaryRenderer('categorical');
-    (<OverviewColumn>this).setWidthImpl(0); // hide
+    this.setWidthImpl(0); // hide
   }
 
   getValue(row: IDataRow) {

